Autopilot/Common/Common.h

```diff
@@ -57,15 +57,28 @@
 #define PM_SET_ORBIT_GAIN 65
 
 //Structs and typedefs
-typedef struct _waypointWrapper{
-    long double longitude;  //TODO: Longitude and Latitude is bulky. If problems arise, change the format.
+
+/* For reference: 
+ In MPLAB XC 16 compiler:
+ char           : 1 byte
+ int            : 2 bytes
+ long int       : 4 bytes
+ float          : 4 bytes
+ long double    : 8 bytes
+ */
+
+
+/*  WAYPOINT WRAPPER IS USED FOR NETWORKING 
+    DO NOT CHANGE THIS UNLESS YOU KNOW WHAT YOU ARE DOING */
```

Comment:
All multipart command structs should be grouped together in the same file, but that is out of the scope of this PR